pull_request_template.md

```diff
@@ -0,0 +1,45 @@
+# Description
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change. Include if this was a client requested change. Include reference to original request.
+
+
+## Related Issues
+
+Link to any GitHub issues or client requests being addressed in this PR (use "#"):
+- Issue/Request: [#issue-number](link)
+
+
+## Type of change
+
+Please delete options that are not relevant. (All options may be deleted for minor changes)
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce the tests. Please also list any relevant details for your test configuration
+
+- [ ] Test with instructions for reproduction
+- [ ] Test with instructions for reproduction
+
+
+## Screenshots (if applicable)
+Include screenshots that highlight the change, especially for UI/UX updates.
+
+
+## Checklist
+- [ ] I have ensured the code adheres to the coding standards of the project.
+- [ ] My changes require documentation updates and I have included them.
```

Comment:
This one looks optional and it isn't 100 percent clear whether I should check it anyway...or if I can delete it if it isn't related to my PR

Reply:
I'll change the wording of line 36 to include "If".
I think that the checklist is a nice reminder to me of the expectations for pull requests and sets a nice precedent for any new people coming to the codebase.